sdk/core/core-amqp/src/requestResponseLink.ts

```diff
@@ -215,16 +215,20 @@ export class RequestResponseLink implements ReqResLink {
    * @param connection - The amqp connection.
    * @param senderOptions - Options that must be provided to create the sender link.
    * @param receiverOptions - Options that must be provided to create the receiver link.
+   * @param options - Optional parameters that can be used to affect this method's behavior.
+   *    For example, `abortSignal` can be passed to allow cancelling an in-progress `create` invocation.
    * @returns Promise<RequestResponseLink>
    */
   static async create(
     connection: Connection,
     senderOptions: SenderOptions,
-    receiverOptions: ReceiverOptions
+    receiverOptions: ReceiverOptions,
+    options: { abortSignal?: AbortSignalLike } = {}
```

Comment:
Nit: Since we have 2 other "options" here, consider naming this `createOptions`?

Reply:
Updated!